### PR TITLE
Add omni

### PR DESCRIPTION
**Pull request by jgarte**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,6 +200,7 @@ A curated list of awesome [Nim](https://nim-lang.org) frameworks, libraries and
 ## Audio
 
 * [paramidi](https://github.com/paranim/paramidi) - A Nim library for making MIDI music. [Unlicense]
+* [omni](https://vitreo12.github.io/omni/) - A DSL for low-level audio programming. [MIT]
```
Comment (xflywind) [inline README.md:203]: please use github links.

Comment (jgarte) [reply README.md]: done! :)

**Comment by jgarte** [comment]
I'm curious does the github web interface have a way for you to squash my commits? I'm using the web interface to submit this.

**Comment by xflywind** [comment]
never mind, I always use squash merge. And thanks for your contribution!

**Pull request merged**